### ISSUE TITLE
RPC: get_content: invalid character '\\x03' in string literal

### ISSUE THREAD

**Issue by tchap** [steemit/steem#189]
Hi,

When calling `get_content` with author `kubrickkk` and permlink `lykke`, I get `invalid character '\\x03' in string literal`.

When trying to access https://steemit.com/@kubrickkk I get `500 Internal Server Errror` with `can't parse steemd's response`.

**Comment by tchap** [comment]
Here is the transaction: https://steemd.com/tx/efee34dc0a9b64d2bae50854c07fdb09ea0a0249

**Comment by arhag** [comment]
@xeroc's steem Python libs also fail when trying to retrieve that post.

The `json.loads` function throws an exception because it was unable to parse the resulting UTF-8 string passed back by steemd as a valid JSON since it does not like that `\x03` (end-of-text) character.

I was able to find a solution/workaround in that particular case by modifying how the Python function `json.loads` was called in xeroc's python-graphenelib libraries. [This line](https://github.com/xeroc/python-graphenelib/blob/246a27e4d1c371f92e518ff91cfcc3450a6f79ce/grapheneapi/graphenewsrpc.py#L339) was replaced by:
```
            ret = json.loads(reply, strict=False)
```

After making that change, I was able to use python-steemlib to retrieve the content of that post. Here is a relevant snippet of the returned string (as printed out to the terminal using Python's pprint function):
```
         'Lykke team\n'
         'Please feel free to contact us to make \x03a suggestion, get '
         'information, or leave \x03a comment: lykke@lykkex.com',
```

The Python [docs](https://docs.python.org/3/library/json.html) for the json module say:
> If strict is false (True is the default), then control characters will be allowed inside strings. Control characters in this context are those with character codes in the 0-31 range, including '\t' (tab), '\n', '\r' and '\0'.

I'm not sure if the appropriate solution is to do a similar sort of thing when handling UTF-8 string including control characters in your Go client, as well as whatever code steemd.com and steemit.com are using to parse the returned string from steemd (because those two websites also have errors when trying to view that particular post), or if the problem should be solved within the steemd codebase by sanitizing the title, body, and json_metadata content to filter out the problematic control characters before returning it in the serialized JSON message over websockets.

I should also mention that cli_wallet does not appear to have a problem dealing with the output from steemd in this case, because I was able to use the `get_state` command in cli_wallet to get and print out to the terminal the contents of that post correctly.
